### ISSUE TITLE
Unable to save image in avif format

### ISSUE THREAD

**Issue by hieudole** [kleisauke/net-vips#125]
Hi,

I'm using NetVips Sample project and I try to save an image in avif format but it says "unable to call VipsForeignSaveHeifFile
heifsave: Unsupported compression".

What package do I need to install in the project in order to make it support avif format?

Thank you!

**Comment by kleisauke** [comment]
Maybe you forgot to set the `compression` to `Enums.ForeignHeifCompression.Av1`? This works for me:
```csharp
using var im = Image.NewFromFile("lichtenstein.jpg", access: Enums.Access.Sequential);
im.Heifsave("lichtenstein.avif", q: 50, compression: Enums.ForeignHeifCompression.Av1, speed: 6, strip: true);
```

Note that since commit https://github.com/libvips/libvips/commit/8c4486327e3a8b7d855c2a996b866df36b0ea0c1 (available in libvips 8.11) this would be automatically set for `*.avif` file suffixes.

**Comment by hieudole** [comment]
Thanks. It works great!
